org.eclipse.ice.commands/src/main/java/org/eclipse/ice/commands/LocalCopyFileCommand.java

```diff
@@ -73,11 +72,14 @@ protected CommandStatus run() {
 		// Try to copy from source to destination, overwriting if the file already
 		// exists at the destination. If it can't, complain.
 		try {
+			status = CommandStatus.RUNNING;
 			Files.copy(source, destination.resolve(source.getFileName()), REPLACE_EXISTING);
 		} catch (IOException e) {
+			status = CommandStatus.FAILED;
 			e.printStackTrace();
```

Comment:
Remove or log.

Reply:
See above comment, I went through and made sure every exception was logged.